# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/gfxPlatformGtk.cpp
# Commit: 6d7485c81e90
# Full Hash: 6d7485c81e90e2d2e70dffc5014bcaaf34b18a87
# Author: Robert Mader <robert.mader@posteo.de>
# Date: 2022-06-21 15:52:43
# Description:
#   Bug 1758473 - Move VA-API test into glxtest, r=stransky
#   
#   Some VA-API drivers are so broken that trying to use them
#   crashes Firefox. This is nothing entirely new for GPU drivers
#   and which is why we have `glxtest`.
# ==============================================================================

diff -r 54aff5bcc909 -r 6d7485c81e90 gfx/thebes/gfxPlatformGtk.cpp
--- a/gfx/thebes/gfxPlatformGtk.cpp	Tue Jun 21 08:19:37 2022 +0000
+++ b/gfx/thebes/gfxPlatformGtk.cpp	Tue Jun 21 08:45:27 2022 +0000
@@ -67,7 +67,6 @@
 #  include <gdk/gdkwayland.h>
 #  include "mozilla/widget/nsWaylandDisplay.h"
 #  include "mozilla/widget/DMABufLibWrapper.h"
-#  include "mozilla/widget/VAAPIUtils.h"
 #  include "mozilla/StaticPrefs_widget.h"
 #endif
 
@@ -260,13 +259,6 @@
     feature.ForceDisable(FeatureStatus::Unavailable, "Requires EGL",
                          "FEATURE_FAILURE_REQUIRES_EGL"_ns);
   }
-
-  if (feature.IsEnabled()) {
-    if (!VAAPIIsSupported()) {
-      feature.ForceDisable(FeatureStatus::Failed, "Failed to configure",
-                           failureId);
-    }
-  }
 #else
   feature.DisableByDefault(FeatureStatus::Unavailable,
                            "Wayland support missing",